conf/unloader.cfg

```diff
@@ -13,8 +13,12 @@ dir_location=/var/spool/apel/
 # VStarRecords
 table_name = VJobRecords
 
-# Only for job records and summaries
+# Only used for job and storage accounting.
```

Comment:
Is this option useable for summary records? The change makes that less clear.

Reply:
My reasoning was that its used for job accounting regardless of whether you are sending summaries or individual job records.

I could change it to:
`#  Only used when sending job records, job summaries or storage records.`

Reply:
Yes, that's good.